changelog/1287.doc.rst

```diff
@@ -0,0 +1,3 @@
+Changed the color of in-line literals to be consistent with the
+`Web Content Accessibility Guidelines 2 Level AA Conformance
+<https://www.w3.org/WAI/WCAG2AA-Conformance>`__ for contrast.
```

Comment:
Is there a specific section in the WCAG you can link to?  It would be good to link to the actual inline literal guidelines.

Reply:
Done.